### DISCUSSION TITLE
Tautomer enumeration rules in the RDKit code--trying to find

### DISCUSSION THREAD

**Discussion by bertiewooster** [rdkit/rdkit#7435] [General] [answered]
I believe I once found a list of tautomer transforms as reaction SMARTS (or perhaps SMIRKS) in the RDKit code, but cannot locate it now. The [reference to the paper “Tautomerism in Large Databases”](https://github.com/rdkit/rdkit/blob/d315e246e14637ddad07090164fdb72dc11348f3/Code/GraphMol/MolStandardize/Wrap/Tautomer.cpp#L377-L379) mentions enumeration rules but they do not seem to be in that file. Is there such a list of tautomer transforms?

**Comment by greglandrum** [comment, marked as answer]
@bertiewooster the tautomers transforms are defined here: https://github.com/rdkit/rdkit/blob/master/Code/GraphMol/MolStandardize/TautomerCatalog/tautomerTransforms.in

**Reply by bertiewooster** [reply]
Great, thanks @greglandrum!
